packages/react-native-bridge/android/build.gradle

```diff
@@ -184,57 +184,52 @@ if (buildGutenbergMobileJSBundle) {
     def bundleName = 'index.android.bundle'
     def jsRootDir = '../../../..'
 
-    task bundleUpToDateCheck {
-        description("Checks if the inputs to the javascript bundle and the bundle itself are unchanged. \
+	task bundleUpToDateCheck {
```

Comment:
Is the whitespace change intentional? 🤔
Can we revert it back if not?

Reply:
> Is the whitespace change intentional? 🤔

Not intentional. Weirdly that difference doesn't show up in my IDE or vim when I look at the file. Seems it is a tabs versus spaces issue. Not sure how it got introduced. 😕 

> Can we revert it back if not?

👍  Definitely, done in 90aa339.

Reply:
Sorry Matt, looks like more lines need the same treatment to remove the added whitespace?

Here https://github.com/WordPress/gutenberg/blob/90aa33903d1d398f1f4c15e53f55d9a5f1c8dc14/packages/react-native-bridge/android/build.gradle#L191-L210, and here https://github.com/WordPress/gutenberg/blob/90aa33903d1d398f1f4c15e53f55d9a5f1c8dc14/packages/react-native-bridge/android/build.gradle#L220-L232?

Reply:
Thanks for catching those. No idea why Android Studio has started insisting on tabs instead of spaces despite the fact that I have it set up to use spaces. Anyway, I'm pretty sure I've got them all fixed in 857f407.